src/components/mapPageComponents/mobileMapPage/index.tsx

```diff
@@ -22,20 +20,17 @@ const MobileMapPage: React.FC<MobileMapPageProps> = ({
   blocks,
   neighborhoods,
   children,
-}) => {
-  return (
-    <>
-      <MainContent>
-        <MapLayout>
-          <Content>
-            <MapView blocks={blocks} neighborhoods={neighborhoods} />
-          </Content>
-          <Footer>{children}</Footer>
-        </MapLayout>
-      </MainContent>
-      <MobileFooter />
-    </>
-  );
-};
+}) => (
+  <>
+    <MainContent>
+      <MapLayout>
+        <Content>
+          <MapView blocks={blocks} neighborhoods={neighborhoods} />
+        </Content>
+        {children}
```

Comment:
❓ no footer?

Reply:
When I add the footer on the mobile map page, it adds padding. This is desirable on the landing page but not on the reservations page. Therefore I removed it from the mobile map page and included it separately in the landing page but not the reservations page.